uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -69,4 +69,22 @@ protected function cleanup() {
     }
   }
 
+  private function createUserAndActivities() {
```

Comment:
I was definitely getting errors that the option value didn't exist. It can happen if there are some changes in the database, for example I was doing some tests about what would happen if `activity_type` option group didn't exist. This test will fail then. But maybe this is a step too far, since we can't protect against manual database manipulation. I'll remove this function.